2024-09-06 14:05:13,079 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-09-06 14:05:13,083 INFO supervisord started with pid 1
2024-09-06 14:05:14,086 INFO spawned: 'laravel' with pid 7
2024-09-06 14:05:15,088 INFO success: laravel entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-09-06 14:05:15,225 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139667278298768 for <Subprocess at 139667291826384 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:05:15,225 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139667278298768 for <Subprocess at 139667291826384 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:09:42,364 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139667278298768 for <Subprocess at 139667291826384 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:09:42,365 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139667278298768 for <Subprocess at 139667291826384 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:09:42,365 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139667278298768 for <Subprocess at 139667291826384 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:10:27,417 WARN received SIGQUIT indicating exit request
2024-09-06 14:10:27,417 INFO waiting for laravel to die
2024-09-06 14:10:27,427 INFO stopped: laravel (exit status 0)
2024-09-06 14:10:43,206 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-09-06 14:10:43,210 INFO supervisord started with pid 1
2024-09-06 14:10:44,212 INFO spawned: 'laravel' with pid 8
2024-09-06 14:10:44,876 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139824836143504 for <Subprocess at 139824840965392 with name laravel in state STARTING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:10:45,878 INFO success: laravel entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-09-06 14:10:49,879 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139824836143504 for <Subprocess at 139824840965392 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:10:49,879 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139824836143504 for <Subprocess at 139824840965392 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:10:49,880 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139824836143504 for <Subprocess at 139824840965392 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:12:45,434 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139824836143504 for <Subprocess at 139824840965392 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:12:45,447 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139824836143504 for <Subprocess at 139824840965392 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:12:45,447 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139824836143504 for <Subprocess at 139824840965392 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:13:21,489 WARN received SIGQUIT indicating exit request
2024-09-06 14:13:21,490 INFO waiting for laravel to die
2024-09-06 14:13:21,497 INFO stopped: laravel (exit status 0)
2024-09-06 14:13:57,917 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-09-06 14:13:57,920 INFO supervisord started with pid 1
2024-09-06 14:13:58,923 INFO spawned: 'laravel' with pid 7
2024-09-06 14:13:59,658 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state STARTING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:13:59,658 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state STARTING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:00,660 INFO success: laravel entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-09-06 14:14:24,182 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:24,182 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:24,183 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:24,183 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:24,183 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:24,183 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139721188039760 for <Subprocess at 139721172411088 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:27,185 WARN received SIGQUIT indicating exit request
2024-09-06 14:14:27,186 INFO waiting for laravel to die
2024-09-06 14:14:27,190 INFO stopped: laravel (exit status 0)
2024-09-06 14:14:31,758 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-09-06 14:14:31,760 INFO supervisord started with pid 1
2024-09-06 14:14:32,763 INFO spawned: 'laravel' with pid 7
2024-09-06 14:14:33,424 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state STARTING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:33,425 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state STARTING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:34,426 INFO success: laravel entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-09-06 14:14:34,926 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:34,926 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:34,927 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:34,927 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:34,927 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:14:34,927 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:15:20,952 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:15:20,952 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:15:20,952 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:15:20,952 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
2024-09-06 14:15:20,952 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140650537273296 for <Subprocess at 140650519657872 with name laravel in state RUNNING> (stdout)> (<class 'OSError'>:[Errno 29] Illegal seek [/usr/lib/python3/dist-packages/supervisor/supervisord.py|runforever|218] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|handle_read_event|276] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|record_output|210] [/usr/lib/python3/dist-packages/supervisor/dispatchers.py|_log|179] [/usr/lib/python3/dist-packages/supervisor/loggers.py|info|327] [/usr/lib/python3/dist-packages/supervisor/loggers.py|log|345] [/usr/lib/python3/dist-packages/supervisor/loggers.py|emit|227] [/usr/lib/python3/dist-packages/supervisor/loggers.py|doRollover|264])
